pkg/log/logger.go

```diff
@@ -172,3 +172,15 @@ func (l *Logger) Log(args ...any) {
 func mapToHclogLevel(level Level) hclog.Level {
 	return hclog.Level(int(hclog.Error) - int(level))
 }
+
+func (l *Logger) AddAndDeleteResource(resource string) {
+	l.Warnf("%s is marked for addition and deletion", resource)
+}
+
+func (l *Logger) DeleteNonExistentResource(resource string) {
+	l.Warnf("%s is marked for deletion but does not exist", resource)
+}
+
+func (l *Logger) AddDuplicateresource(resource string) {
+	l.Warnf("%s is marked for addition but already exists", resource)
+}
```

Comment:
Can we define this function closer to where it's being used, and pass in the logger as a parameter? If we start adding a bunch of functions in this file for every command that could get out of hand after a while.